homeassistant/components/sensor/mychevy.py

```diff
@@ -154,13 +158,20 @@ def async_update_callback(self):
         """Update state."""
         if self._car is not None:
             self._state = getattr(self._car, self._attr, None)
+            for attr in self._extra_attrs:
+                self._state_attributes[attr] = getattr(self._car, attr, None)
```

Comment:
We usually don't include the item at all if the value is `None`.

Reply:
ack